lib/s3middleware/objectLegalHold.js

```diff
@@ -67,7 +65,7 @@ function _validateLegalHold(parsedXml) {
  * @param {string} xml - xml body to parse and validate
  * @param {object} log - werelogs logger
  * @param {function} cb - callback to server
- * @return {function} - callback with legal hold boolean on success,
+ * @return {Error|object} - calls callback with legal hold status on success,
```

Comment:
I still think the @return here is incorrect as nothing is actually returned we just call the callback.

Reply:
Yeah seems like across similar methods returning callback it's mostly listed as function so I updated to function. Let me know if you have any other suggestion instead of that. (I thought of `undefined` also but it didn't seem quite right either.)

Reply:
I am fairly certain it is `undefined`, you can add a test case or adjust an existing test case to see what is returned. But if you do not return anything in JS you will inherently get undefined as a return value. Most languages do this as far as I know.

Reply:
Updated - didn't realize `undefined` was what you meant initially.